js/form_question_dropdown.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/* global sortable, GlpiFormQuestionTypeSelectable */
+
+class GlpiFormQuestionTypeDropdown extends GlpiFormQuestionTypeSelectable {
+
+    /**
+     * Create a new GlpiFormQuestionTypeSelectable instance.
+     *
+     * @param {string} inputType
+     * @param {JQuery<HTMLElement>} container
+     */
+    constructor(inputType = null, container = null) {
+        super(inputType, container);
+
+        this._container.on('sortupdate', () => this.#handleSortableUpdate());
+
+        this._container.closest('[data-glpi-form-editor-question-details]')
+            .find('div[data-glpi-form-editor-specific-question-options]')
+            .find('input[data-glpi-form-editor-original-name=is_multiple_dropdown]')
+            .on('change', (event) => {
+                this._inputType = event.target.checked ? 'checkbox' : 'radio';
+                this.#updateInputType(this._inputType);
```

Comment:
Cahnging the `_inputType` property should be done in the `#updateInputType()` method
```suggestion
                this.#updateInputType(event.target.checked ? 'checkbox' : 'radio');
```